crowdsourcing/viewsets/task.py

```diff
@@ -58,22 +58,18 @@ def create(self, request, *args, **kwargs):
             return Response(serializer.errors,
                             status=status.HTTP_400_BAD_REQUEST)
 
+    def update(self, request, *args, **kwargs):
```

Comment:
if this is a detail route then self.get_object() will do and update should happen in the serializer